### ISSUE TITLE
Mode 8 can't unload digestate into tip triggers

### ISSUE THREAD

**Issue by Blackmankot** [Courseplay/courseplay#2629]
Здравствуйте! Транспорт digestat снова не работает к сожалению. Если вы используете FS17_kotteUniversalPack пкг 1.2.0.0. Что погрузка и выгрузка не работает вообще. Если вы используете подобный танк от самой игры, загрузки, а выгрузка не работает. Режим транспортировки 8.
https://fs17.lt/kotte-universal-pack-v-1-2/
Видео - https://yadi.sk/i/vRGUWDLt3TCo7M
И конечно, спасибо за работу!
Channel 23
1. ![2018-03-10_23-38-05](https://user-images.githubusercontent.com/26131714/37243926-5730c556-24bc-11e8-98a0-c5256cda2abd.jpg)
[log.txt](https://github.com/Courseplay/courseplay/files/1799517/log.txt)

2. ![2018-03-10_23-44-47](https://user-images.githubusercontent.com/26131714/37244005-7c395df8-24bd-11e8-8ccf-1a7f90739e56.jpg)
[log.txt](https://github.com/Courseplay/courseplay/files/1799528/log.txt)

3. ![2018-03-10_23-51-17](https://user-images.githubusercontent.com/26131714/37244055-251d7e4a-24be-11e8-9f1c-f62d1c143ba8.jpg)
[log.txt](https://github.com/Courseplay/courseplay/files/1799531/log.txt)

**Comment by pops64** [comment]
I never setup mode 8 to handle digesate which is what your using. This will go under has a mod issue because on standard maps there are no tip triggers which accept digesate. Only field tanks and sprayers accept digesate and you can only sell slurry and sell triggers are tip triggers ussally

**Comment by Blackmankot** [comment]
I think if the Kotte pack can not load the digestat into the BHA - this is not a problem. A similar tank from the game is perfectly loaded and that's enough. It is sad that she now does not unload the 8 mode. And in previous versions everything worked perfectly! I would be grateful if you can restore the unloading.

**Comment by pops64** [comment]
Nothing has changed on CP's in regards to that. So either the mod or map has been updated so something broke.

**Comment by Blackmankot** [comment]
I checked, if you use Pak Kotte to version 1.2.0.0., then downloading to BGA is excellent! With the unloading, too, everything is fine, transport in new versions of the course stops earlier at the waiting point. Therefore, the transport was standing, and the trigger did not see the tank. I wrote down a new course with a waiting point beyond the unloading trigger. And it all worked.
But the version of the Kotte pack is from 1.2.0.0. does not work at all ...
